doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -232,97 +280,169 @@ some special operations to happen on them internally.
-ROS Control tab can be used to auto generate simulated controllers to actuate the joints of your robot. This will allow us to provide the correct ROS interfaces MoveIt.
+MoveIt requires trajectory controllers with a FollowJointTrajectoryAction interface for 
+executing planned trajectories. This interface sends the generated trajectory to the robot ROS 2 Controllers.
 
-* Click on the *ROS Control* pane selector.
+MoveIt Controllers pane can be used to auto-generate the controllers to be used by MoveIt controller manager. 
+Ensure that the controller names match those configured in the previous  ROS 2 controller step. 
+The user interface for this step is similar to the previous one.
 
-.. image:: setup_assistant_panda_ros_control.png
+.. image:: moveit_controllers/setup_assistant_moveit_controllers.png
    :width: 700px
 
-* Click on *Add Controller* and you should see the following screen:
+Add the arm MoveIt controllers
 
-* We will first add Panda arm position controller
+* Click on the MoveIt Controllers pane selector.
 
-* Enter *Controller Name* as **arm_position_controller**
+* Click on Add Controller to create a new arm controller.
 
-* Choose **position_controllers/JointPositionController** as the controller type
+* Enter Controller Name as panda_arm_controller
 
-* Next you have to choose this controller joints, you can add joints individually or add all the joints in a planning group all together.
+* Choose **FollowJointTrajectory** Controller Type. 
 
-* Now, click on Add Planning Group Joints.
+* Choose the controller joints with the panda_arm planning group.
 
-.. image:: setup_assistant_panda_ros_control_create.png
+.. image:: moveit_controllers/setup_assistant_panda_arm_moveit_controller_type.png
    :width: 700px
 
-* Choose panda_arm planning group to add all the joints in that group to the arm controller.
 
-.. image:: setup_assistant_panda_ros_control_add_joints.png
+Add the hand MoveIt controllers
+
+* Follow the same steps for the arm, but choose **Gripper Command** Controller Type. 
+  
+* Add the joints using the hand move group and save the controller.
+
+.. image:: moveit_controllers/setup_assistant_hand_moveit_controller_type_gripper.png
    :width: 700px
+  
+After completing the previous steps, the MoveIt Controllers list for the arm and hand should appear as follows.
 
-* Click *Save* to save the selected controller.
+.. image:: moveit_controllers/setup_assistant_moveit_controllers_done_gripper.png
+   :width: 700px
+
+Step 11: Perception
+-------------------
+
+The Perception tab in the Setup Assistant is used to configure the settings 
+for 3D sensors used by the robot. These settings are saved in a YAML configuration file named **sensors_3d.yaml**.
+
+In case of **sensors_3d.yaml** was not needed, choose **None** and proceed to the next step.
+
+.. image:: perception/setup_assistant_panda_3d_perception.png
+   :width: 700px
 
-Step 11: Add Author Information
+To generate **point_cloud** configuration parameters:
```

Comment:
To generate **point_cloud** configuration parameters see the following example:
NOTE: this configuration is not valid for the Panda robot arm because it does not have a `head_mounted_kinect` camera.

Reply:
Thank you for highlighting this point! I updated the perception step accordingly.